### DISCUSSION TITLE
Add sync button for the exchange rate

### DISCUSSION THREAD

**Discussion by Jim8Knopf** [enrique-lozano/Monekin#202] [Ideas / New features]
Enhancement I'd love  
---

Currently you need to enter the exchange rate manually.

Possible solutions
---

I think the best thing would be to add a syc button that syncs the selected currencies wit the internet. Then you can still adjust it if you want manually.

**Comment by enrique-lozano** [comment]
Hi! First of all, thank you for using the app and taking your time to write your improvement ideas here. 

As I mention [here](https://github.com/enrique-lozano/Monekin/discussions/160#discussioncomment-9306081), this is planned and it will be a great feature. I've to study yet the public APIs available and how to implement this in term of UX. Any ideas are welcome. Hope that I can launch something regarding this feature in Monekin v8. 

I will probably add an issue with this feature request.

Again, thanks a lot!

**Reply by Jim8Knopf** [reply]
Thank you very much and I'll reed it the next time more carefully.
